scalableminds/webknossos
# Fix frontend search for objects with null values

There was a regression in the frontend search for keys whose value was an object which contained `null` or `undefined` values - leading to a full site crash :see_no_evil: 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open the tasks view and select a project with tasks.
- Enter a random search term in the search field at the very top which doesn't find any tasks. -> The site should not crash. (it does on master)

------
